whois-api/src/test/java/net/ripe/db/whois/api/rest/ZonemasterTestIntegration.java

```diff
@@ -54,7 +54,21 @@ public void start_domain_test_when_dsrdata_digest_contains_spaces() {
 
         assertThat(response.getResult(), is("b3a92c89c92414ed"));
     }
+    @Test
+    public void start_domain_test_with_final_dot() {
+        zonemasterDummy.whenThen(ZonemasterRequest.Method.START_DOMAIN_TEST.getMethod(), "{\"jsonrpc\":\"2.0\",\"id\":4,\"result\":\"b3a92c89c92414ed\"}\n");
+        final RpslObject domainObject = RpslObject.parse(
+                "domain:    1.0.10.in-addr.arpa.\n" +
```

Comment:
It should be zonemasterDummy.whenThen("1.0.10.in-addr.arpa", "{\"jsonrpc\":\"2.0\",\"id\":4,\"result\":\"b3a92c89c92414ed\"}\n");
So basically you are telling the dummy server to return a response when a request for domain name without trailing dot comes. This way the actual domain object is with trailing dot (as you craeted a domain name with this) but when sending request to zonemaster there is no trailing dot